packages/labs/eleventy-plugin-lit/README.md

```diff
@@ -46,14 +46,15 @@ npm i @lit-labs/eleventy-plugin-lit
 
 ### Register plugin
 
-Edit your `.eleventy.js` config file to register the Lit plugin:
+Edit your `.eleventy.cjs` config file to register the Lit plugin:
```

Comment:
I would just keep this as `.js`, since that's the default extension most people will have when using eleventy.